### PR TITLE
UG > PagNav: Misleading sentence

### LINKED ISSUE #2412: UG > PagNav: Misleading sentence
image

This second sentence might mislead readers into thinking that PageNavs cannot be used in normal pages.

### PR DESCRIPTION
**Pull request by KevinEyo1**
**What is the purpose of this pull request?**

- [X] Documentation update
- [ ] Bug fix
- [ ] Feature addition or enhancement
- [ ] Code maintenance
- [ ] DevOps
- [ ] Improve developer experience
- [ ] Others, please explain:

**Overview of changes:**
Fixes #2412 
Updated documentation to avoid misleading wording.

**Anything you'd like to highlight/discuss:**

**Testing instructions:**

**Proposed commit message: (wrap lines at 72 characters)**
UG: update wording of UG

Wording might mislead readers into thinking that PageNavs 
cannot be used in normal pages.

Updating the wording will allow readers to know that 
individual pages can also use PageNavs.

Let's update the wording to be clearer and more explicit.

---

**Checklist:** :ballot_box_with_check:

- [ ] Updated the documentation for feature additions and enhancements
- [ ] Added tests for bug fixes or features
- [X] Linked all related issues
- [X] No unrelated changes 

---

**Reviewer checklist:**

Indicate the [SEMVER](https://semver.org/) impact of the PR:
- [ ] Major (when you make incompatible API changes)
- [ ] Minor (when you add functionality in a backward compatible manner)
- [ ] Patch (when you make backward compatible bug fixes)

At the end of the review, please label the PR with the appropriate label: `r.Major`, `r.Minor`, `r.Patch`.

Breaking change release note preparation (if applicable):
- To be included in the release note for any feature that is made obsolete/breaking

> Give a brief explanation note about:
>   - what was the old feature that was made obsolete
>   - any replacement feature (if any), and
>   - how the author should modify his website to migrate from the old feature to the replacement feature (if possible).

### CODE REVIEW

File: docs/userGuide/syntax/pageNavigationMenus.md [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 <div id="content">
 
-**A _Page Navigation Menu_ (==_pageNav_ for short==)  a list of the current page's headings.** Page navigation menus are only available for use in [layouts]({{baseUrl}}/userGuide/tweakingThePageStructure.html#layouts).
+**A _Page Navigation Menu_ (==_pageNav_ for short==) displays a list of the current page's headings.** Page navigation menus are typically used in [layouts]({{baseUrl}}/userGuide/tweakingThePageStructure.html#layouts), but they can also be utilized in individual pages.
```
Comment (damithc) [inline docs/userGuide/syntax/pageNavigationMenus.md:7]: 'used in layouts' still doesn't sound right. It's more like set/configured/controlled via the layouts mechanism.

**Review by yucheng11122017 — APPROVED** [review]
LGTM! Thank you @KevinEyo1

File: docs/userGuide/syntax/pageNavigationMenus.md [hunk 2]
```diff
@@ -4,7 +4,7 @@
 
 <div id="content">
 
-**A _Page Navigation Menu_ (==_pageNav_ for short==)  a list of the current page's headings.** Page navigation menus are only available for use in [layouts]({{baseUrl}}/userGuide/tweakingThePageStructure.html#layouts).
+**A _Page Navigation Menu_ (==_pageNav_ for short==) displays a list of the current page's headings.** Page navigation menus are typically configured in [layouts]({{baseUrl}}/userGuide/tweakingThePageStructure.html#layouts), but they can also be utilized in individual pages.
```
Comment (itsyme) [inline docs/userGuide/syntax/pageNavigationMenus.md:7]:
```suggestion
**A _Page Navigation Menu_ (==_pageNav_ for short==) displays a list of the current page's headings.** Page navigation menus are typically configured in [layouts]({{baseUrl}}/userGuide/tweakingThePageStructure.html#layouts), but can also be utilized in individual pages.
```

**Comment by itsyme** [comment]
LGTM! Thanks for the work @KevinEyo1

**Review by itsyme — APPROVED** [review]

**Pull request merged**